lib/ansible/modules/system/systemd.py

```diff
@@ -283,6 +291,13 @@ def parse_systemctl_show(lines):
                 k = None
     return parsed
 
+def is_running_in_chroot():
+    try:
+        s1 = os.stat('/')
+        s2 = os.stat('/proc/1/root')
```

Comment:
shouldn't this be '/proc/1/root/.' ?
also it requires root to establish chroot status, if running as normal user you are 'always' in chroot.

looking at adding this as a 'fact' that the module can import, but that needs to be more generic (this one is linux specific) and work w/o root if possible

Reply:
https://github.com/ansible/ansible/pull/30525